chart/charts/crds/values.yaml

```diff
@@ -0,0 +1,4 @@
+jaeger:
+  # -- Install Jaeger CRDs
+  enabled: false
```

Comment:
Should be `enabled: true` by default. The dependent chart should set it to false, as you have set it already.